ecosystem/sep-0024.md

```diff
@@ -1158,6 +1159,22 @@ Each object in the `transactions` array should have the following fields:
 | `amount`  | string | The amount sent back to the user for the payment identified by `id`, in units of `amount_in_asset`.                                                                                                                                                                  |
 | `fee`     | string | The amount charged as a fee for processing the refund, in units of `amount_in_asset`.                                                                                                                                                                                |
 
+### Fee Details Object Schema
+
+| Name        | Type   | Description                                                                                                                                                           |
+| ----------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `total`     | string | The total amount of fee applied.                                                                                                                                      |
+| `asset`     | string | The asset in which the fee is applied, represented through the [Asset Identification Format](#asset-identification-format).                                           |
```

Comment:
This should link to `sep-0038.md#asset-identification-format`. `#asset-identification-format` does not exist in this file. The same applies to SEP-6.